doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -117,40 +116,42 @@ you will need to rebuild Chapel from source in a different configuration.
       # re-build Chapel
       make
 
-      # make check is available but optional
-      make check
-
       # compile a sample program
-      chpl -o hello examples/hello.chpl
+      chpl -examples/hello3-datapar.chpl
 
       # run the sample program
-      ./hello
+      ./hello3-datapar
 
-   Note that the environment settings from ``util/setchplenv.bash`` will not persist beyond this terminal session.
-   You can choose to source ``setchplenv.bash`` whenever you want to use relevant commands.
-   If you want these environment settings to persist for future terminal sessions,
-   copy the commands from the Recommended Settings in :ref:`chplenv.rst <readme-chplenv.recommended_settings>` into your ``~/.bashrc`` file.
-   You can also store Chapel configuration settings in a :ref:`chplconfig <readme-chplenv.chplconfig>` file instead of your ``~/.bashrc``.
+   If you run into any portability issues, please see
+   :ref:`readme-bugs`:
 
-   See :ref:`chplenv.rst <readme-chplenv>` for a complete description of
-   Chapel's configuration variables, what they mean, and how they
-   can be set.
+
+Using Chapel in a Different Shell / Session
+-------------------------------------------
+
+Note that in both the Quickstart and preferred modes above, any
+environment settings made by ``setchplenv.bash`` will not persist
+beyond your current shell / terminal session.  To use Chapel from a
+different shell or terminal, you will need to either re-``source`` the
+``setchplenv.bash`` script that you used when building Chapel or set
+up your environment to support additional shells automatically.  See
+:ref:`using-chapel-in-other-sessions` for details.
 
 
 Using Chapel in Multi-Locale Mode
 ---------------------------------
 
-All of the instructions above describe how to run Chapel programs
-in a single-locale (shared-memory) mode. To run using multiple
-locales (distributed memory), please refer to
-:ref:`multilocale.rst <readme-multilocale>`.
+All of the instructions above describe how to run Chapel programs in a
+single-locale (shared-memory) mode. To run using multiple locales
+(distributed memory), please refer to :ref:`readme-multilocale`.
 
-Performance
------------
 
-If you plan to do performance studies of Chapel programs, be sure to use the
-full-featured version from using-a-more-full-featured-chapel_ above and see
-https://chapel-lang.org/performance.html for performance tips.
+Notes on Performance
+--------------------
+
+If you plan to do performance studies of Chapel programs, be sure to
+use the full-featured version above and see
```

Comment:
Except for when you want performance on a machine with multiple nodes?  Also, we should consider using the term node when we use the term locale, since people just getting started with Chapel won't know the correspondence there.

Reply:
> Except for when you want performance on a machine with multiple nodes?

I think of this comment as being orthogonal to the multiple node question (where I think of its main purpose as being "don't do performance studies with the quickstart settings" and from there, you're welcome to do either single- or multiple-node performance studies).  (?)

Reply:
Previously the "full-featured version" was implicitly referring to single node.  If you made that explicit, then this is fine.